profiler/options.go

```diff
@@ -224,6 +224,7 @@ func WithProfileTypes(types ...ProfileType) Option {
 		for k := range cfg.types {
 			delete(cfg.types, k)
 		}
+		cfg.addProfileType(MetricsProfile) // always report metrics
```

Comment:
Product decision here, but if a customer opts out of metrics collection, parts of the UI won't work as expected